test/sequential/test-fs-watch-system-limit.js

```diff
@@ -36,14 +36,27 @@ function spawnProcesses() {
 
 spawnProcesses();
 
+setTimeout(() => {
+  if (!finished) {
+    const timeoutStream = new stream.PassThrough();
+    timeoutStream.write('Error: timeout');
+    timeoutStream.end();
+    timeoutStream.pipe(gatherStderr);
+
+    finished = true;
+    processes.forEach((proc) => proc.kill());
+  }
+}, timeout);
+
 let accumulated = '';
 gatherStderr.on('data', common.mustCallAtLeast((chunk) => {
   accumulated += chunk;
   if (accumulated.includes('Error:') && !finished) {
     assert(
       accumulated.includes('ENOSPC: System limit for number ' +
                            'of file watchers reached') ||
-      accumulated.includes('EMFILE: '),
+      accumulated.includes('EMFILE: ') ||
+      accumulated.includes('Timeout'),
```

Comment:
This spells `Timeout` with a capital T, the text above has a lowercase t, so they don’t match … is that intentional?

Reply:
unintentional, thank you